clients/metadata/metadata_cassandra.go

```diff
@@ -3703,6 +3798,81 @@ func (s *CassandraMetadataService) readConsumerGroupExtentsHelper(request *m.Rea
 	return query.Iter(), nil
 }
 
+// ReadConsumerGroupExtentsLite returns the list all extents mapped to
+// the given consumer group. This API only returns a few interesting
+// columns for each extent in the result. For detailed info about
+// extents, see ReadConsumerGroupExtents
+func (s *CassandraMetadataService) ReadConsumerGroupExtentsLite(ctx thrift.Context, request *m.ReadConsumerGroupExtentsLiteRequest) (*m.ReadConsumerGroupExtentsLiteResult_, error) {
+
+	if request.GetMaxResults() < 1 {
+		return nil, errPageLimitOutOfRange
+	}
+
+	filterLocally := false
+	if request.IsSetStatus() && request.GetStatus() != m.ConsumerGroupExtentStatus_OPEN {
+		filterLocally = true
+	}
+
+	sql := sqlCGGetAllExtentsLite
+	if !filterLocally && request.IsSetStatus() {
+		sql += fmt.Sprintf(" AND %s=%d", columnStatus, request.GetStatus())
+	}
+
+	qry := s.session.Query(sql).Consistency(s.lowConsLevel).Bind(request.GetConsumerGroupUUID())
+	if request.GetMaxResults() > 0 {
```

Comment:
isn't this check redundant? since we enforce 'GetMaxResults' to be at least '1' above?